post-office-stub/src/data/postOfficeResponse/postResponse429.ts

```diff
@@ -0,0 +1,9 @@
+export const POST_REPONSE_429 = {
+	"errors": [
+		{
+			"error": "TooManyRequestsError",
+			"message": "Exceeded send limits (LIMIT NUMBER) for today",
```

Comment:
Does this mock need this?

Reply:
These error responses were requested by @Z-Mohamoud, have changed the error messages to be more generic